applications/asset_tracker_v2/src/modules/ui_module.c

```diff
@@ -26,6 +26,13 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_UI_MODULE_LOG_LEVEL);
 
+/* Macro redefinition used to expose internal variables in module test. */
+#if defined(CONFIG_UNITY) || defined(CONFIG_ZTEST)
```

Comment:
```suggestion
#if defined(CONFIG_UNITY) 
```
It is better to fail when if using ZTEST in the future so that the developer is aware of this.